testing/trino-product-tests/src/main/java/io/trino/tests/kafka/TestKafkaAvroReadsSmokeTest.java

```diff
@@ -205,20 +183,51 @@ public String toString()
         }
     }
 
+    private static void createAvroTable(String schemaPath, String tableName, String topicName, Map<String, Object> record, MessageSerializer messageSerializer)
+    {
+        try {
+            String schema = Files.readString(new File(schemaPath).toPath());
+            createAvroTable(new TestingAvroSchema(schema, ImmutableList.of(), ImmutableList.of()), tableName, topicName, record, messageSerializer);
+        }
+        catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    private static void createAvroTable(TestingAvroSchema schema, String tableName, String topicName, Map<String, Object> record, MessageSerializer messageSerializer)
+            throws IOException
```

Comment:
maybe just add `throws Exception` and allow tests to throw `Exception`. No boilerplate with try-catch and rethrow as runtime

Reply:
I prefer the way with UncheckedIO, but this is irrelevant really here.